config.ru

```diff
@@ -1,6 +1,6 @@
-RACK_ENV = ENV['RACK_ENV'] ||= 'development' unless defined?(RACK_ENV)
```

Comment:
this should remain RACK_ENV I think. Line 3 on the other hand should be DEPLOY env because we want Sentry to run on the Staging and Production environments in AWS